.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request
+description: Suggest an idea for the project
+labels:
+  - "enhancement"
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing feature requests before submitting your own."
+  - type: textarea
+    id: enhancement-summary
+    attributes:
+      label: Enhancement Summary
+      description: |
+        - Briefly describe the enhancement.
+        - Explain how the proposed enhancement helps overcome a limitation of LMMS you are currently facing.
```

Comment:
How is this second point different from the "justification" section that comes next?

Reply:
On second thought, it's not, so I merged both sections together.